kv/range_cache.go

```diff
@@ -33,6 +33,10 @@ import (
 // RangeCache.
 type rangeCacheKey proto.Key
 
+func (a rangeCacheKey) String() string {
+	return proto.Key(a).String()
+}
```

Comment:
`proto.Key` also implements the `Formatter` interface: https://github.com/cockroachdb/cockroach/blob/master/proto/data.go#L160

This is what forces quoted printing no matter what verb is used.